packages/backend/src/managers/modelsManager.spec.ts

```diff
@@ -119,9 +122,14 @@ const configurationRegistryMock: ConfigurationRegistry = {
   getExtensionConfiguration: vi.fn(),
 } as unknown as ConfigurationRegistry;
 
+let modelHandlerRegistry: ModelHandlerRegistry;
+
 beforeEach(() => {
   vi.resetAllMocks();
-  taskRegistry = new TaskRegistry({ postMessage: vi.fn().mockResolvedValue(undefined) } as unknown as Webview);
+  taskRegistry = new TaskRegistry({ postMessage: () => Promise.resolve() } as unknown as Webview);
+  modelHandlerRegistry = new ModelHandlerRegistry({
+    postMessage: vi.fn().mockReturnValue(Promise.resolve()),
+  } as unknown as Webview);
```

Comment:
The webview seems to be shared here, but there two different way to mock the postMessage?
- `postMessage: () => Promise.resolve()`
-  `postMessage: vi.fn().mockReturnValue(Promise.resolve())`

We could simplify by adding a
```ts
const WEBVIEW_MOCK: Webview = {
  postMessage: vi.fn(),
} as unknown as Webview;

beforeEach(() => {
  vi.mocked(WEBVIEW_MOCK.postMessage).mockResolvedValue(undefined);
})
```